tokenizer.go

```diff
@@ -3,7 +3,8 @@ package tokenizers
 // TODO packaging: how do we build the rust lib for distribution?
 
 /*
-#cgo LDFLAGS: ${SRCDIR}/libtokenizers.a -ldl -lm -lstdc++
+#cgo tokenizers_srcdir_relative LDFLAGS: ${SRCDIR}/libtokenizers.a -ldl -lm -lstdc++
+#cgo !tokenizers_srcdir_relative LDFLAGS: /usr/lib/libtokenizers.a -ldl -lm -lstdc++
```

Comment:
why are you picking `/usr/lib/libtokenizers.a` over package relative `libtokenizers.a`?

Reply:
I mentioned that in my comment, by making the build flag name "tokenizers_srcdir_relative" as you suggest, package relative therefore becomes the non-default option. If you prefer to keep the current default behaviour we need to change that flag name to something like "tokenizers_srcdir_static"?  Do you have a preferred name?